Exercises/Exercise-1/main.py

```diff
@@ -12,8 +14,19 @@
 
 
 def main():
-    # your code here
-    pass
+    for uri in download_uris:
+        download_and_unzip(uri)
```

Comment:
Add `try except `block with logging what went wrong on failure and success message otherwise